bld/CLMBuildNamelist.pm

```diff
@@ -5081,6 +5079,29 @@ sub setup_logic_misc {
 
 #-------------------------------------------------------------------------------
 
+sub setup_logic_prigent_roughness {
+  #
+  # The Prigent roughness stream data set read in if needed
+  #
+  my ($opts, $nl_flags, $definition, $defaults, $nl) = @_;
+  my $var = "use_prigent_roughness";
+  my $dust_emis_method = remove_leading_and_trailing_quotes( $nl->get_value('dust_emis_method') );
+  add_default($opts, $nl_flags->{'inputdata_rootdir'}, $definition, $defaults, $nl, $var,
+              'dust_emis_method'=>$dust_emis_method );
+  my $use_prigent = $nl->get_value($var);
+  if ( &value_is_true($use_prigent) ) {
+     if ( $dust_emis_method ne "Leung_2023" ) {
```

Comment:
Add a note about why this is a warning as it could be used for other purposes in the future.

Reply:
Make it explicit that it won't do anything and won't be read -- unless Leung_2023 is active. But, it also wont' hurt anything.